web/src/util/Types.ts

```diff
@@ -1 +1,4 @@
+import { Ref } from "@vue/composition-api";
+
 export type SortingFunction<T> = (a: T, b:T) => number;
+export type MaybeRef<T> = T | Ref<T>;
```

Comment:
Your editor seems to be omitting newlines on the end of a file, can you change this?

Maybe we should also add an `.editorconfig` to define our preferences.